MAUI/ImageEditor/save.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Save in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about save support in the Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Save in .NET MAUI ImageEditor (SfImageEditor)
+
+The Image Editor control in .NET MAUI allows you to save the edited image as PNG and JPEG.
+
+To save the modified image, you can use the `Save` method, which accepts parameters such as file name, file type, file path, and image size. The supported file types for saving are PNG and JPEG. You can also save the image by clicking the save button on the toolbar.
+
+{% tabs %}
+
+{% highlight C# tabtitle="C#" %}
+
+              imageEditor.Save(ImageFileType.Png, "D:\\Syncfusion\\Pictures", "Syncfusion");
+
+{% endhighlight %}
+
+{% endtabs %}
+
```

Comment:
The saved image will be added to the device for each platform in the following locations:

Windows, MacCatalyst and iOS:
System.Environment.SpecialFolder.MyPictures
Link: https://learn.microsoft.com/en-us/dotnet/api/system.environment.specialfolder?view=net-7.0

Windows device Location: “C:\Users\your name\Pictures

MacCatalyst device Location : /Users/your name/Documents/Pictures
iOS device location: /Photos/Pictures

Android
API 29 and above: Pictures folder of Android.Provider.MediaStore.IMediaColumns.RelativePath 

Link - https://developer.android.com/reference/android/provider/MediaStore.MediaColumns#RELATIVE_PATH

API 28 and below: 
Android.Provider.MediaStore.Images.Media.ExternalContentUri

Link: https://developer.android.com/reference/android/provider/MediaStore.Images.Media#EXTERNAL_CONTENT_URI

Device location: \Internal storage\Pictures

Note: For android should include permission in AndroidManifest.xml folder

Link: https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/appmodel/permissions?tabs=android

  <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
  <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />

Reply:
Added a content